compiler-rt/lib/interception/interception_win.cpp

```diff
@@ -907,6 +912,11 @@ static size_t GetInstructionSize(uptr address, size_t* rel_offset = nullptr) {
   return 0;
 }
 
+// Unfortunately size_t is not known when compiling asan_allocator.cpp
```

Comment:
I guess the problem with size_t is only in the header file, interception_win.h? If so, can we use SIZE_T only there, and size_t everywhere else? (Or are they not compatible?)

We could also use `unsigned` for the test function instead of `size_t`..

Reply:
Sorry I have replaced just the SIZE_T in the test, unfortunately not yet in interception_win.cpp.
Now the only SIZE_T should be in the header.